packages/amplify-appsync-simulator/src/velocity/util/elasticsearch-helper.ts

```diff
@@ -5,15 +5,14 @@ class ElasticsearchHelper {
 
   private static readonly ERROR_FORMAT: string = 'Could not construct an Elasticsearch Query DSL from {0} and {1}';
 
-
   /**
    * This method is used by the ModelTransformUtils.
    *
    * For Example, the following filter parameter:
    *
    * filter: {
    *   title: {
-   *     eq: "hihihi",
+   *     eq: "hi",
```

Comment:
:-) I'm pretty sure the intention here was to demonstrate wildcard. perhaps we should invent some string that's bigger than `hi` but matches `h*i` or leave it as is. Or perhaps `hi hi hi`.